index.d.ts

```diff
@@ -0,0 +1,23 @@
+import type {Plugin} from 'rollup'
+/**
+ * globals is a moduleId/variableName map
+ * or provide a function that takes the moduleId and returns the variableName
+ */
+type ModuleNameMap = Record<string, string> | ((id: string | number) => string)
+
+type ExternalGlobalsOptions = {
+  /**
+   * [include] is an array of glob patterns. If defined, only matched files would be transformed.
+   */
+  include?: Array<string>
+  /**
+   * [exclude] is an array of glob patterns. Matched files would not be transformed.
+   */
+  exclude?: Array<string>
+  /**
+   * [dynamicWrapper] is used to specify dynamic imports. Below is the default.
+   */
+  dynamicWrapper?: ((id: string | number) => string)
```

Comment:
`dynamicWrapper` accepts a variable name and returns an expression. It should be better with:
```ts
(variableName: string) => string
```